app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -29,6 +29,13 @@ import MetadataInput from "./MetadataInput";
 const map = _fp.map.convert({ cap: false });
 
 class MetadataManualInput extends React.Component {
+  // List as of 2020-01-08.
+  INSECT_GENOME_IDS = [
```

Comment:
Did you consider including this info in the DB schema somehow (host genome model)? We might start missing updating this when we add new insect genomes.

Reply:
Yes, I considered it... and I think that it is reasonable to add a column such as `sample_type_taxa_name` to `host_genomes`. However, I decided against it for this PR because:
* we haven't had a chance to play with the suggestions yet and there may be bigger changes that result 
* the hard-coded precedent in the code of `isHuman` 
* the cost of a missed insect is low: the same options would be available in the "ALL" category instead of "SUGGESTED"